unit_tests/test_hydra_sh.py

```diff
@@ -195,6 +195,16 @@ def clean_resources_cmd_docker(self):
         # Command line that should be run in the docker
         return f'clean-resources --post-behavior --test-id {self.test_id}'
 
+    @cached_property
+    def clean_runner_instances_cmd(self):
```

Comment:
I'm not sure those are the best tests to use to cover the functionality

It's probably better to build a specific test suite for it